apps/meteor/app/lib/server/methods/insertOrUpdateUser.ts

```diff
@@ -18,13 +19,14 @@ Meteor.methods<ServerMethods>({
 		methodDeprecationLogger.method('insertOrUpdateUser', '8.0.0');
 
 		check(userData, Object);
+		const userId = Meteor.userId();
 
-		if (!Meteor.userId()) {
+		if (!userId) {
 			throw new Meteor.Error('error-invalid-user', 'Invalid user', {
 				method: 'insertOrUpdateUser',
 			});
 		}
 
-		return saveUser(Meteor.userId(), userData);
+		return saveUser(userId, userData as ISaveUserDataParams);
```

Comment:
I think we should add a validation this is what we're receiving, because the method seems to check only for object.

wdyt? cc @MarcosSpessatto for the wdyt too

Reply:
Agreed

Reply:
The issue is that this would be breaking, wouldn't it? If we're doing that, I believe that should be implemented in another PR
Should we have a task for that, wdyt @MarcosSpessatto ? This method is deprecated, though